### PR TITLE
Add GetQueuesRuntimeInfoAsync to ManagementClient

### PR DESCRIPTION
**Pull request by tmasternak**
This adds `GetQueuesRuntimeInfoAsync` method to `ManagementClient`. Connected to #8434

### CODE REVIEW

**Comment by nemakam** [comment]
@tmasternak 
The change looks good. Only thing is we might have to do the same for topic and subscriptions as well. Need to keep the API layer consistent across the entities. Could you please add that as well?

**Review by nemakam — CHANGES_REQUESTED** [review]
The change looks good. Only thing is we might have to do the same for topic and subscriptions as well. Need to keep the API layer consistent across the entities. Could you please add that as well?

**Comment by tmasternak** [comment]
@nemakam sure. I'll update the PR.

**Comment by tmasternak** [comment]
@nemakam I've extended the PR with async queries for topic and subscription runtime info.

**Comment by jsquire** [comment]
//fyi:  @JoshLove-msft, @ShivangiReja, @ramya-rao-a

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs [hunk 1]
```diff
@@ -90,5 +92,34 @@ static TopicRuntimeInfo ParseFromEntryElement(XElement xEntry)
...
+                        return topicList;
+                    }
+                }
+            }
+            catch (Exception ex) when (!(ex is ServiceBusException))
+            {
+                throw new ServiceBusException(false, ex);
+            }
+
+            throw new MessagingEntityNotFoundException("Topic was not found");
```
Comment (nemakam) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs]: >Topic was not found [](start = 56, length = 19)

nit: No topic was found

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections.Generic;
+
```
Comment (nemakam) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs]: nit: Move inside the namespace to keep it consistent

**Review by nemakam — APPROVED** [review]
The changes look good. There are two nit comments. I'll merge once you've taken care of them.

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs (hunk 2 above)
Comment (nemakam) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs]: Same in other classes

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs (hunk 1 above)
Comment (nemakam) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs]: Same in other classes

Comment (tmasternak) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs]: Fixed

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs (hunk 2 above)
Comment (tmasternak) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs]: Fixed

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs (hunk 1 above)
Comment (nemakam) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs]: Ideally, we would like to have "No topics were found" in collection view, and "topic was not found" when trying to parse a single topic.

Comment (tmasternak) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs]: I've updated the exception messages.

**Comment by nemakam** [comment]
@tmasternak , thanks for the PR

**Pull request merged**